src/components/summary/examples/summary-household/index.njk

```diff
@@ -62,13 +62,14 @@
                                 }
                             ]
                         }
-                    ]
+                    ],
+                    "summaryLink": {
+                        "text": "Add someone to this household",
+                        "url": "#0",
+                        "dataQa": "add-item-link"
+                    }
                 }
             ]
         }
     ]
 }) }}
-
-<div class="u-mb-l u-pt-s u-bt">
```

Comment:
I think this div should have a class, then apply top padding in the css like we do for the summary rows.  Then the service can use utility classes on `summary` to add the desired bottom margin to the component

Reply:
I don't think it needs a class to do that now its been integrated into the component but I have moved the mb to the `summary` and set it in the example which I think will achieve the same result

Reply:
Don't we usually try to make classes for component elements?  Just makes it more future proof, allowing us to make styling changes in the future.

Reply:
Yeah makes sense